spring-kafka/src/main/java/org/springframework/kafka/core/reactive/ReactiveKafkaConsumerTemplate.java

```diff
@@ -146,10 +146,22 @@ public ReactiveKafkaConsumerTemplate(ReceiverOptions<K, V> receiverOptions) {
 		return doOnConsumer(consumer -> consumer.position(partition));
 	}
 
+	/**
+	 * Get the committed {@link OffsetAndMetadata} for the partition.
+	 * @param partition the partition.
+	 * @return the {@link OffsetAndMetadata}.
+	 * @deprecated in favor of {@link #committed(Set)}.
+	 */
+	@Deprecated
+	@SuppressWarnings("deprecation")
 	public Mono<OffsetAndMetadata> committed(TopicPartition partition) {
 		return doOnConsumer(consumer -> consumer.committed(partition));
```

Comment:
Can we just build a `Set` here and call that new method instead of `@SuppressWarnings` ?

Reply:
That's not so easy because we'd have to convert a Mono of Map to a Mono of the single value.

I think this is easier.